editor/src/components/navigator/navigator-utils.ts

```diff
@@ -179,7 +179,20 @@ export function getNavigatorTargets(
     }
   }
 
-  const canvasRoots = MetadataUtils.getAllStoryboardChildrenPathsUnordered(metadata)
+  function getCanvasRoots(trees: ElementPathTree[]): ElementPath[] {
```

Comment:
I wonder if this code should become an implementation of `getAllStoryboardChildrenPathsOrdered`.